src/extension.ts

```diff
@@ -72,9 +72,17 @@ export function activate(extensionContext: vscode.ExtensionContext) {
     let standaloneCompileCommand = vscode.commands.registerCommand('shader-toy.createPortableGlslPreview', () => {
         shadertoyManager.createPortablePreview();
     });
+    let pausePreviewsCommand = vscode.commands.registerCommand('shader-toy.pauseGlslPreviews', () => {
+        shadertoyManager.postCommand('pause');
+    });
+    let saveScreenshotCommand = vscode.commands.registerCommand('shader-toy.saveGlslPreviewScreenShots', () => {
```

Comment:
This variable has a different name than the command, screenshot should be plural

Reply:
Little thing but yes.